tests/library_checker_aizu_tests/math/count_paths.test.cpp

```diff
@@ -0,0 +1,87 @@
+#define PROBLEM "https://judge.yosupo.jp/problem/number_of_increasing_sequences_between_two_sequences"
+#include "../template.hpp"
+
+#define const ;
+#define mod mod_not_using_one
+#include "../../../library/math/n_choose_k/grow.hpp"
+#undef const
+#undef mod
+
+#define mod mod_not_using_two
+#define modpow modpow_not_using
+#include "../../../kactl/content/number-theory/ModPow.h"
+#undef mod
+#undef modpow
+
+ll modpow(ll b, ll e) {
+  ll ans = 1;
+  for (; e; b = b * b % 998'244'353, e /= 2)
+    if (e & 1) ans = ans * b % 998'244'353;
+  return ans;
+}
+
+#include "../../../library/math/count_paths/count_paths_triangle.hpp"
+
+ll count_between_two_sequences(vl a, vl b) {
+  int n = sz(a);
+  vl dp(b[0] - a[0]);
+  dp[0] = 1;
+  for (int i = 0; i < n;) {
+    int j = i;
+    {
+      vl h(b[i] - a[i]);
+      for (int num = a[i]; num < b[i]; num++) {
+        while (j < n && a[j] <= num) {
+          j++;
+        }
+        h[num - a[i]] = j - i;
+      }
+      assert(sz(dp) <= sz(h));
+      while (sz(dp) < sz(h)) dp.push_back(0);
+      dp = divide_and_conquer(h, dp);
+    }
+    {
+      vl h(j - i);
+      for (int k = i; k < j; k++) {
+        h[k - i] = b[k] - b[i];
+      }
+      if (h.back() == 0) break;
+      dp = divide_and_conquer(h, dp);
+      if (j < n) {
+        dp = vl(begin(dp) + (a[j] - b[i]), end(dp));
+      }
+    }
+    i = j;
+  }
+  return dp.back();
+}
+
+int main() {
+  cin.tie(0)->sync_with_stdio(0);
+  mod_not_using_one = 998'244'353;
```

Comment:
why are you setting a mod that you aren't using? let's rename this it's not as clear as it could be imo